writeengine/server/we_dmlcommandproc.h

```diff
@@ -78,8 +78,8 @@ class WE_DMLCommandProc
   EXPORT uint8_t rollbackVersion(messageqcpp::ByteStream& bs, std::string& err);
   EXPORT uint8_t processBatchInsert(messageqcpp::ByteStream& bs, std::string& err,
                                     ByteStream::quadbyte& PMId);
-  EXPORT uint8_t processBatchInsertBinary(messageqcpp::ByteStream& bs, std::string& err,
-                                          ByteStream::quadbyte& PMId);
+  //EXPORT uint8_t processBatchInsertBinary(messageqcpp::ByteStream& bs, std::string& err,
```

Comment:
We need this? If not plz remove.

Reply:
Let's left it as it might be needed by the bulk insertion API.

Reply:
Done. I have uncommented back this change.